src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala

```diff
@@ -108,8 +163,43 @@ class TasksController(
     case _ => None
   }
 
+  @SuppressWarnings(Array("all")) // async/await
+  private def doKillTasks(toKillFuture: Future[Map[PathId, Seq[Instance]]], tasksIdToAppId: Map[Id, PathId], wipe: Boolean)(implicit identity: Identity): Future[Seq[EnrichedTask]] = async {
```

Comment:
Could you add docs? It is not quite clear from the method names how this differs to `doKill` and `deleteTasks`.

Reply:
Added docs and did some renaming. Let me know if it did not help.